package-lock.json

```diff
@@ -2089,12 +2090,20 @@
       "version": "1.5.0",
       "resolved": "https://registry.npmjs.org/@colors/colors/-/colors-1.5.0.tgz",
       "integrity": "sha512-ooWCrlZP11i8GImSjTHYHLkvFDP48nS4+204nGb1RiX/WXYHmJA2III9/e2DWVabCESdW7hBAEzHRqUn9OUVvQ==",
-      "dev": true,
-      "optional": true,
       "engines": {
         "node": ">=0.1.90"
       }
     },
+    "node_modules/@dabh/diagnostics": {
```

Comment:
I am not sure if you can commit all this.  Eddie can confirm on this?

Reply:
Thanks Ravi 👍  if there are changes to the `package.json` file, then we will need changes to the lock file

Reply:
We can discuss more on our LinkFree office hours community call tonight